counterfab/src/main/java/com/andremion/counterfab/CounterFab.java

```diff
@@ -133,6 +139,9 @@ public CounterFab(Context context, AttributeSet attrs, int defStyleAttr) {
         }
 
         mCirclePaint.setColor(ta.getColor(R.styleable.CounterFab_badgeBackgroundColor, defaultBadgeColor));
+        if (ta.hasValue(R.styleable.CounterFab_badgePosition)) {
+            badgePosition = ta.getInt(R.styleable.CounterFab_badgePosition, 0);
```

Comment:
You could have just this line below without the `if` condition:
````kotlin
badgePosition = ta.getInt(R.styleable.CounterFab_badgePosition, RIGHT_TOP_POSITION);
````

Reply:
done